Dockerfile

```diff
@@ -1,17 +1,30 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
 
 ENV TZ="America/New_York"
 ENV repo_dir="/ot-efr32"
+WORKDIR ${repo_dir}
 
 # Install packages
 RUN apt-get update && \
       apt-get -y install sudo tzdata && \
       rm -rf /var/lib/apt/lists/*
 
 # Copy scripts
-RUN mkdir -p ${repo_dir}/third_party/silabs/slc
-COPY ./script ${repo_dir}/script
-COPY ./openthread/script ${repo_dir}/openthread/script
+COPY ./openthread/script ./openthread/script
```

Comment:
Is this repeated?

Reply:
I meant `COPY ./openthread/script ./openthread/script`

Reply:
That's not how the `COPY` command in Dockerfile works. Please see https://docs.docker.com/engine/reference/builder/#copy

It copies `./openthread/script` from the build context into the docker image's filesystem